### ISSUE TITLE
Tag OSS products

### ISSUE THREAD

**Issue by fmerian** [agamm/awesome-developer-first#87]
why do you think if we add a `open-source` tag to OSS products? 👍 👎 ?

**Comment by keichinger** [comment]
Personally, I'd love this additional information. Yes, there are a lot of services out there that are paid, but some of them even offer a free/OSS self-hosted version in addition to their SaaS offering. The amount of people that are looking for this additional on-premise offering may be a small, though.

**Comment by agamm** [comment]
YES! How do you think about adding it? Can you give an example screenshot?

**Comment by fmerian** [comment]
> YES! How do you think about adding it? Can you give an example screenshot?

I was thinking of adding inline code at the end of a line:
```
`open-source`
```
![awesome-developer-first-oss+edit](https://github.com/agamm/awesome-developer-first/assets/1890087/e2388cc2-8fd6-48cc-93bf-1cb3854b5e52)
![awesome-developer-first-oss+preview](https://github.com/agamm/awesome-developer-first/assets/1890087/8f2ef978-55d6-474e-9fdf-221dc291c62a)

**Comment by Jogai** [comment]
Better with a badge: (https://flat.badgen.net/github/license/micromatch/micromatch)

**Comment by fmerian** [comment]
I like badges, too. I'm wondering if verifying licenses wouldn't be an extra, unnecessary layer of work when reviewing new entries -- wdyt @agamm?

**Comment by agamm** [comment]
Maybe just "Github" badge?

**Comment by fmerian** [comment]
> Maybe just "Github" badge?

oh I like this idea

**Comment by agamm** [comment]
Yep looks good.

**Comment by Jogai** [comment]
But stars don't say anything about how open something is...

**Comment by fmerian** [comment]
> But stars don't say anything about how open something is...

IMHO we don’t need this level of information. we just want to tag COSS products. let’s keep it simple :)

**Comment by agamm** [comment]
I think that defining what is open-source license/legal-wise is very hard to do at scale and I don't think we also want to take liability on that.
If that's important for someone, having the badge link to the repo will be great (they could check the license there). Stars also show some part of "awesomeness" so it's good.

**Comment by agamm** [comment]
One counterpoint is to not put stars so people understand that this is a product, not "opensource" awesome list.

**Comment by Jogai** [comment]
> > But stars don't say anything about how open something is...
> 
> IMHO we don’t need this level of information. we just want to tag COSS products. let’s keep it simple :)

Just a badge makes it simple. And in my opinion stars don't say that much, see https://opensource.stackexchange.com/questions/5110/github-stars-is-a-very-useful-metric-but-for-what & https://dagster.io/blog/fake-stars

**Comment by fmerian** [comment]
that's right. `# of contributors` may be more relevant, with the right level of awesomeness.

I'll go with this:

```
![GitHub contributors](https://img.shields.io/github/contributors-anon/github/docs?style=flat-square&logo=github&labelColor=%230D1117&color=%23161B22)
```

preview:

![GitHub contributors](https://img.shields.io/github/contributors-anon/github/docs?style=flat-square&logo=github&labelColor=%230D1117&color=%23161B22)

**Comment by fmerian** [comment]
some reflections from #105:

- **should we reconsider GitHub Stars vs. Contributors?** first reaction was to check the # of Stars to appreciate how awesome a product is. 

- **should we set an amount of Stars to consider a product as "awesome"**? for example, a GitHub project must have above 5k Stars to be defined as "popular" in [simple-icons](https://github.com/simple-icons/simple-icons/) ([direct link to contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-an-icon))
